internal/testfixtures/datastore.go

```diff
@@ -119,10 +126,23 @@ func StandardDatastoreWithSchema(ds datastore.Datastore, require *require.Assert
 	validating := NewValidatingDatastore(ds)
 
 	allDefs := []*core.NamespaceDefinition{UserNS, FolderNS, DocumentNS}
+	caveats := []*core.CaveatDefinition{{
+		Name:                 "testcaveat",
+		SerializedExpression: []byte{},
+	}}
+
+	_, _ = ds.ReadWriteTx(ctx, func(ctx context.Context, rwt datastore.ReadWriteTransaction) error {
```

Comment:
I see you kept this outside of the transaction below to make sure it does not fail for stores that do not support this just yet.

Reply:
Yeah, for now